tests/integration/staking/keeper/slash_test.go

```diff
@@ -271,7 +274,7 @@ func TestSlashWithUnbondingDelegation(t *testing.T) {
 	assert.NilError(t, f.stakingKeeper.SetUnbondingDelegation(f.sdkCtx, ubd))
 
 	// slash validator for the first time
-	f.sdkCtx = f.sdkCtx.WithBlockHeight(12)
+	f.sdkCtx = f.sdkCtx.WithHeaderInfo(header.Info{Height: 12})
```

Comment:
should we change this later on? Make it a task at the end to swap out the `sdk.context` in the tests?

Reply:
yea, im not sure how we should be setting it tbh, need to think through that